governator-jersey/src/main/java/com/netflix/governator/guice/jersey/GovernatorJerseySupportModule.java

```diff
@@ -0,0 +1,80 @@
+package com.netflix.governator.guice.jersey;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.netflix.governator.guice.jetty.GovernatorServletContainer;
+import com.netflix.governator.providers.AdvisableAnnotatedMethodScanner;
+import com.netflix.governator.providers.ProvidesWithAdvice;
+import com.sun.jersey.api.core.DefaultResourceConfig;
+import com.sun.jersey.api.core.PackagesResourceConfig;
+import com.sun.jersey.api.core.ResourceConfig;
+import com.sun.jersey.guice.spi.container.servlet.GuiceContainer;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+import javax.servlet.ServletContext;
+
+/**
+ * Module enabling support for Jersey configuration via advisable bindings for {@link DefaultResourceConfig}.
+ * This module provides an alternative Guice integration based on jersey-guice {@link GuiceContainer} but with 
+ * less opinions, such as automatically adding any bound class with {@literal @}Path as a resources.
+ * 
+ * Applications are expected to customize {@link DefaultResourceConfig} to provide a whitelist of
+ * root resources to expose.  This can be done by adding the following {@link @Advises} method.
+ * 
+ * <pre>{@code
+@Advises
+@Singleton
+@Named("governator")
+UnaryOperator<DefaultResourceConfig> adviseWithMyApplicationResources() {
+    return defaultResourceConfig -> {
+        defaultResourceConfig.getClasses()
+            .addAll(Arrays.asList(MyApplicationResource.class));
+        return defaultResourceConfig;
```

Comment:
Users may still want to have the ability to scan their (specifically declared) packages. Looks like a way to do so would be something like
```
DefaultResourceConfig scanned = new PackagesResourceConfig("com.my.specific.package");
defaultResourceConfig.getClasses()
    .addAll(scanned.getClasses());
```
Perhaps provide documentation/easier api for this use case?